Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -380,8 +425,8 @@ When determining the <code>deaBan</code>, we first divide the maximum and the
 minimum deviations of measurement
 from the setpoint by the <code>r</code>, respectively.
 We then calculate the absolute values of the two deviations.
-After, we set the <code>deaBan</code> to be half of the smaller one among those
```

Comment:
`When determining the <code>deaBan</code>, we first divide the maximum and the minimum difference of measurement from the setpoint by the typical range of control error <code>r</code>, then find the absolute value of the two deviations. The <code>deaBan</code> can be set as the half of the smaller one between the two absolute deviations.`

Reply:
Updated the description accordingly.